src/components/article/Article.js

```diff
@@ -75,30 +81,24 @@ export default function Article(props) {
 	const onArticleClick = (evt) => {
 		if ((!evt.target.classList.contains("article__saved")) && (!evt.target.classList.contains("article__delete")) && (!evt.target.classList.contains("article__saved_active"))) {
 			window.open(article.url, '_blank', 'noopener,noreferrer');
-		// } else {
-		// 	if ((evt.target.classList.contains("article__saved")) || (evt.target.classList.contains('article__delete')) || (evt.target.classList.contains('article__saved_active'))) {
-		// 		// saveArticleClick(evt);
-		// 	}
 		}
 	}
 
 	React.useEffect(() => {
 		if (isHomePage) {
 			if (article && currentUser) {			// eslint-disable-next-line
 				currentUser.savedArticles.map((savedArticle) => {
-					if (article.url === savedArticle.url) {
-						setDidSave(true);
-					} else {
-						setDidSave(false);
+					if (article.url == savedArticle.url) {
+						setIsSaved(true);
 					}
 				});
 			}
 		}											// eslint-disable-next-line
-	}, [currentUser, didSave]);
+	}, []);
 
 	return (
-		<li className="article" id={article._id ? article._id : ''} key={article.id} onClick={onArticleClick}>
-			<button className={`${isHomePage ? "article__saved" : "article__delete "}${didSave ? '_active' : ''}`} onMouseOut={handleMouseOut} onMouseOver={handleMouseOver} onClick={saveArticleClick} />
+		<li className="article" id={article._id ? article._id : ''} key={key} onClick={onArticleClick}>
```

Comment:
`key={key}`

please, delete it

* when I search anything I get errors in the console about `key`. See here https://disk.yandex.ru/i/SXVUuwNU-KnkgA 


Attribute `key` is needed only when you render an array of components. You shouldn’t put it inside any component initially